src/test/java/com/aws/greengrass/disk/spool/ConnectionFake.java

```diff
@@ -0,0 +1,1912 @@
+/*
```

Comment:
Do we need this whole class? It is pretty insane...

Reply:
You may find lombok's delegate annotation helpful.

Reply:
Yeah I felt insane writing it.... that annotation looks very promising, taking a look

Reply:
Dropped it by 1k lines so that's good :P